public/index.html

```diff
@@ -1,24 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8"/>
-    <link href="%PUBLIC_URL%/favicon.ico" rel="icon"/>
-    <meta content="width=device-width, initial-scale=1" name="viewport"/>
-    <meta content="#000000" name="theme-color"/>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Hast du hier ne umgekehrte alphabetische Anordnung oder was nimmst du da?

Reply:
Kann Dir nicht folgen. Als ich in den Branch gewechselt bin, wurden mir diese Files als merge commits angezeigt. Da ich keine entscheidenden Unterschiede feststellen konnte, habe ich sie einfach committet.

Reply:
Nee, hab bei den Props zumindest keine feste Reihenfolge. Hat den Anschein gemacht als ob das deine IDE macht. Aber dann passt das. Imports sortier ich alphabetisch und nach relativen und library imports